packages/comby/comby.1.2.2/opam

```diff
@@ -0,0 +1,63 @@
+opam-version: "2.0"
+maintainer: "rvantonder@gmail.com"
+authors: "Rijnard van Tonder"
+homepage: "https://github.com/comby-tools/comby"
+bug-reports: "https://github.com/comby-tools/comby/issues"
+dev-repo: "git+https://github.com/comby-tools/comby.git"
+license: "Apache-2.0"
+build: [
+    [
+      "dune"
+      "build"
+      "-p"
+      name
+      "-j"
+      jobs
+      "@install"
+      "@runtest" {with-test}
+    ]
+]
+depends: [
+  "dune" {>= "2.7.0"}
+  "ocaml" {>= "4.08.1"}
+  "angstrom" {>= "0.15.0"}
+  "conf-autoconf"
+  "conf-gmp"
+  "conf-libpcre"
+  "conf-m4" {os = "linux"}
+  "conf-pkg-config"
+  "conf-sqlite3"
+  "conf-zlib" {os = "linux"}
+  "hack_parallel" {arch != "arm32" & arch != "arm64"}
+  "conduit-lwt-unix"
+  "core"
+  "lwt"
+  "lwt_react"
+  "opium"
+  "cohttp-lwt-unix"
+  "parany"
+  "patience_diff" {>= "v0.14" & < "v0.15"}
+  "ppxlib"
+  "ppx_deriving"
+  "ppx_deriving_yojson" {>= "3.6.0"}
+  "pcre"
+  "shell"
+  "toml" {>= "6.0.0"}
+  "tls"
+  "bisect_ppx" {with-test & dev & >= "2.5.0"}
+]
+x-ci-accept-failures: ["oraclelinux-8"] # libev not available by default
```

Comment:
I looked at some of the other `opam` files and I think adding this makes sense? Seemed like previous `oraclelinux-8` failures happened due to missing `libev` on the system but I see no reason it wouldn't work if `libev` were installed.

Reply:
> oraclelinux-8,  comby.1.2.2 (failed: The compilation of conf-libev failed at "/usr/bin/sh ./build.sh".)

whelp, I guess that didn't work

Reply:
Does adding `conf-libev` to the dependency list not work?

Reply:
🤦 how did I miss this

Reply:
Updated, thanks for catching @avsm

Reply:
whoops, I should remove the `x-ci-accept-failures` part too

Reply:
easy to miss, there are many odd quirks with handling external dependencies across so many distributions. Glad it works now!